CONTRIBUTING.md

```diff
@@ -39,7 +39,10 @@ Keeping the `master` releasable means that changes merged to it need to be:
 1. Clone the repository.
 2. Run `yarn`. (Installs dependencies and links packages in the workspace.)
 3. Run `git config commit.template .github/.COMMIT_TEMPLATE` (Sets you up with our commit message template)
-4. Run `yarn start` in the root folder. (Start watching and automatically re-building packages when there are new changes.)
+4. Run `yarn build` to build all packages before start watching.
```

Comment:
did you find that this was necessary before running `yarn start`?

Reply:
I've done some clean installs of the repository with the original instructions, without `yarn build` I couldn't start the test project `my-app`.

Reply:
`yarn start` doesn't build the packages.